### PR TITLE
MTA-2205 add login to registry.redhat.io

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [MTA-2205](https://issues.redhat.com/browse/MTA-2205)

### PREVIEW

* [2.1. Installing the CLI](https://deploy-preview-849--windup-documentation.netlify.app/docs/cli-guide/master/#installing-cli-tool_cli-guide)

### CODE REVIEW

**Comment by anarnold97** [comment]
@dymurray && @abrugaro - please can I ask for a review. thanks

File: docs/topics/installing-cli-tool.adoc [hunk 1]
```diff
@@ -41,14 +42,27 @@ To install using the downloadable `.zip` file:
...
 When you encounter `<{ProductShortName}_HOME>` in this guide, replace it with the actual path to your {ProductShortName} installation.
 
-////
+
 [id="installing-using-podman_{context}"]
 == Installing the {CLINameTitle} using Podman
 
+.Prerequisites
+
+* Red Hat Container Registry Authentication for `registry.redhat.io`. Red Hat distributes container images from  registry.redhat.io, which requires authentication. For more details, see link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication].
```
Comment (abrugaro) [inline docs/topics/installing-cli-tool.adoc:51]: There are two whitespaces between "from" and "registry.redhat.io", is that expected?

The same thing happens in line 15

**Review by abrugaro — APPROVED** [review]
LGTM, minor comment

File: docs/topics/installing-cli-tool.adoc [hunk 2]
```diff
@@ -12,6 +12,7 @@ You can install the {CLINameTitle} on Linux, Windows, or macOS operating systems
 
 .Prerequisites
 
+* Red Hat Container Registry Authentication for `registry.redhat.io`. Red Hat distributes container images from  registry.redhat.io, which requires authentication. For more details, see link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication].
```
Comment (anarnold97) [inline docs/topics/installing-cli-tool.adoc:15]:
```suggestion
* Red Hat Container Registry Authentication for `registry.redhat.io`. Red Hat distributes container images from registry.redhat.io, which requires authentication. For more details, see link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication].
```

File: docs/topics/installing-cli-tool.adoc (hunk 1 above)
Comment (anarnold97) [inline docs/topics/installing-cli-tool.adoc:51]:
```suggestion
* Red Hat Container Registry Authentication for `registry.redhat.io`. Red Hat distributes container images from registry.redhat.io, which requires authentication. For more details, see link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication].
```

Comment (anarnold97) [reply docs/topics/installing-cli-tool.adoc]: @abrugaro - WOW! You have eyes like a hawk!
Apologies, i was going too quickly and missed it

**Review by dymurray — APPROVED** [review]

**Pull request merged**